api/src/routes/api/classifiers/index.js

```diff
@@ -222,6 +224,9 @@ const classifiersRoutes = async (fastify, opts) => {
         run.percentMinusCurPercent = percent >= 100 ? 0 : percentMinusCurPercent;
         run.classifier = number;
         run.index = index;
+        if (division.startsWith('scsa')) {
+          run.hf = Number(run.stageTimeSecs);
```

Comment:
This is the "adapter" hack that makes everything work in the frontend.

Reply:
Added the comment.  Not sure if there's any meaningful thing we can do here, the best solution to this would be to use immutable types with type specialization in TS, but that refactor is probably a ways away for us.